lib/geoengineer/gps/gps.rb

```diff
@@ -156,6 +156,20 @@ def expanded_hash
 
   def loop_projects_hash(projects_hash, &block)
     projects_hash.each_pair do |project, environments|
+      # If the environments includes _other, pull out its value, and loop over
+      # all other known environments and add it back in, if it doesn't already
+      # exist. This allows _other to be used as a template for cookie cutter
+      # definitions.
+      if environments.key?("_other")
+        other = environments.delete("_other")
+
+        # NOTE: the constants appeared to be the best place to easily get all known environments.
+        all_environments = @constants.constants.keys.reject { |env| env.start_with?('_') }
```

Comment:
Is there a reason you filter by all events starting with `_` instead of explicitly filtering only `_other`? Are there other special environment names we exclude? Or are you thinking of adding more?

Reply:
Yes, global constants will show as in `_global`. It is a special keyword within the constants class.